slices/SocialSharerSlice/index.vue

```diff
@@ -11,6 +12,7 @@
 
 <script setup lang="ts">
 import { getSliceComponentProps } from '@prismicio/vue'
+import { asText } from '@prismicio/helpers'
 
 defineProps(getSliceComponentProps(['slice', 'index', 'slices', 'context']))
 </script>
```

Comment:
Is SocialSharerSlice supposed to be in this PR? I don't think it is used.

Reply:
I had to bring them over to keep the slice machine working properly on this branch. All slices would have to be in all branches and kept updated regardless of if we use them or not.